spec/tools/server_settings_replicator/server_settings_replicator_spec.rb

```diff
@@ -0,0 +1,31 @@
+$LOAD_PATH << Rails.root.join("tools")
+
+require "server_settings_replicator/server_settings_replicator"
+
+describe ServerSettingsReplicator do
+  let(:miq_server) { EvmSpecHelper.local_miq_server }
+  let!(:miq_server_remote) { EvmSpecHelper.remote_miq_server }
+  let(:settings) { {:k1 => {:k2 => {:k3 => 'v3'}}} }
+
+  describe "#replicate" do
+    it "targets only other servers" do
+      miq_server.add_settings_for_resource(settings)
+      expect(described_class).to receive(:copy_to).with([miq_server_remote], settings)
```

Comment:
Is there any reason that we can't actually check that the setting was saved for the other server here?

Why stub out the call to `.copy_to`?

Reply:
wanna check the arguments being passed to it

Reply:
Right, but you're not actually testing that the settings were saved. I'm okay with this either way, I just lean towards not stubbing things if I don't have to.

Reply:
right, that's the thinking
👍

Reply:
I actually wrote my feedback for your followup PR before seeing this. I agree, I think the mock here is hiding a larger issue with your class interface and test boundaries: https://github.com/ManageIQ/manageiq/pull/16023#pullrequestreview-64897892